draft-ietf-scitt-architecture.md

```diff
@@ -80,58 +80,65 @@ venue:
-However, the range of use cases and applications in this document is much broader, which requires much more flexibility in how each Transparency Service implements and operates its Registry.
-Each service may enforce its own policy for authorizing entities to register their Signed Statements on the Transparency Service.
-Some Transparency Services may also enforce access control policies to limit who can audit the full Registry, or keep some information on the Registry encrypted.
-Nevertheless, it is critical to provide global interoperability for all Transparency Services instances as the composition and configuration of involved supply chain entities and their system components is ever-changing and always in flux.
-
-A Transparency Services provides visibility into Signed Statements originally created as Statements and issued as Signed Statements by supply chain entities and their sub-systems.
-These Signed Statements (and corresponding Statement payload) are about the objects produced by supply chain objects: Artifacts.
-A Transparency Service vouches for specific and well-defined metadata about these Artifacts that is captured in Statements.
+However, the range of use cases and applications in this document is much broader, which requires much more flexibility in how each Transparency Service is implemented and operates.
+Each service MAY enforce its own Registration Policy for authorizing entities to register their Signed Statements to the append-only Ledger.
+Some Transparency Services may also enforce role based access control (RBAC) policies limiting who can write, read and audit specific Feeds or the full Registry.
+It is critical to provide interoperability for all Transparency Services instances as the composition and configuration of involved supply chain entities and their system components is ever-changing and always in flux.
+
+A Transparency Services provides visibility into Signed Statements associated with various supply chains and their sub-systems.
+These Signed Statements (and corresponding Statement payload) are about the Artifacts produced by a supply chain.
+A Transparency Service endorses specific and well-defined metadata about these Artifacts that is captured in Statements.
 Some metadata is selected (and signed) by the Issuer, indicating, e.g., "who issued the Statement" or "what type of Artifact is described" or "what is the Artifact's version"; whereas additional metadata is selected (and countersigned) by the Transparency Services, indicating, e.g., "when was the Signed Statement about the Artifact registered in the Registry".
-A Statements payload content typically is opaque to the Transparency Services, if so desired: it is the metadata that must always be transparent in order to warrant trust for later processing.
+The countersignature is often referred to as being notarized.
```

Comment:
```suggestion
Producing a Transparent Statement is also referred to as a form of notarization.
```

Reply:
Added. Note, I'm not using the GithUb commit suggestion as it bypasses signing :(